src/assets/global.scss

```diff
@@ -1,4 +1,5 @@
 @import './variables';
+@import './raleway';
```

Comment:
Ok, now I understand a bit better.

The choice of the font is a design choice.
At the moment, mixing the Kuzzle Design Guidelines with the Material Design Guidelines leads to an ugly result so, IMHO, we should stick with the Material. Currently, the Kuzzle Design Guidelines have been created with print and website in mind, not the BO. To me, it's not the same context.
If we want to conciliate the BO with the Kuzzle Design Guidelines, we should enrich them with this new context (the one of an interaction-intensive application, rather than a mostly read-oriented experience).

Roboto is a great font for the interaction-intensive applications because it is elegant and clear at the same time. It provides great transversal legibility, which is exactly what we need for buttons and labels (which are the main bricks of our UI).
Raleway is great for brand-oriented content, it's elegant but better optimized for blocks of texts rather than buttons and labels. It's not really a "pragmatic" font.

In conclusion, IMHO we should use Roboto, because it is more coherent with Material and better for the user.